docs/developers/java-dev.rst

```diff
@@ -0,0 +1,64 @@
+Java Devolopment
+================
+
+Introduction
+------------
+
+GraphAr Java library based on GraphAr C++ library and an efficient FFI
+for Java and C++ called
+`FastFFI <https://github.com/alibaba/fastFFI>`__.
+
+Source Code Level
+~~~~~~~~~~~~~~~~~
+
+-  Interface
+
+-  Class
+
+-  JNI code
+
+-  C++ source code
+
+Developers only need to write interfaces with annotations. For
+annotation's usage, please refer to
+`FastFFI <https://github.com/alibaba/fastFFI>`__.
+
+FastFFI will automatically generate ``.java`` files which implement
```

Comment:
After the interfaces are ready, the java code for the interfaces and the the C++ code which include JNI code for native methods will be automatically generated by FastFFI.